save-preprocessor/src/main/resources/application-linux.properties

```diff
@@ -0,0 +1 @@
+save.repository=${user.home}/.save-cloud/repositories
```

Comment:
Probably it is a good idea to move it to `application-dev.properties` and remove it from both here and `application-win.properties`

Reply:
what's about mac?

Reply:
Add properties in ```application-dev.properties```, remove from ````application-win.properties``` and ```application-linux.properties```

Reply:
> what's about mac?

Soul Goodman